Chafu/MediaType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Chafu
+{
+	[Flags]
+	public enum MediaType
```

Comment:
There is already a MediaType class here: https://github.com/BruelAndKjaer/Chafu/blob/a51cbbcf0a5c70c0e2d08f7e020129e15fdb3aed/Chafu/BaseAlbumDataSource.cs#L9 use that